src/dash-table/conditional/index.ts

```diff
@@ -29,17 +34,31 @@ export interface IEditableElement {
 }
 
 export type ConditionalBasicFilter = INamedElement & ITypedElement;
-export type ConditionalDataCell = IConditionalElement & IIndexedRowElement & INamedElement & ITypedElement & IEditableElement;
+export type ConditionalDataCell = IActiveElement & IConditionalElement & IIndexedRowElement & INamedElement & ITypedElement & IEditableElement;
 export type ConditionalCell = INamedElement & ITypedElement;
 export type ConditionalHeader = IIndexedHeaderElement & INamedElement & ITypedElement;
 
 function ifAstFilter(ast: QuerySyntaxTree, datum: Datum) {
     return ast.isValid && ast.evaluate(datum);
 }
 
+export function ifColumnActive(condition: IActiveElement | undefined, active: boolean, selected: boolean) {
+    if (!condition) {
+        return true;
+    }
+
+    // Apply active if active, apply selected if selected AND not active (active and selected styling don't overlap)
+    return (condition.is_active === undefined || active === condition.is_active) &&
+        (condition.is_selected === undefined || (selected === condition.is_selected && !active));
+}
+
 export function ifColumnId(condition: INamedElement | undefined, columnId: ColumnId) {
-    return !condition ||
-        condition.column_id === undefined ||
+    if (!condition || condition.column_id === undefined) {
+        return true;
+    }
+
+    return Array.isArray(condition.column_id) ?
+        R.contains(columnId, condition.column_id) :
```

Comment:
```suggestion
        R.includes(columnId, condition.column_id) :
```

`contains` is deprecated (other instances too)

Reply:
https://github.com/plotly/dash-table/pull/729/commits/2af76ce8767edb59be885b108ee0dfab513adf54